src/Command/AllCommand.php

```diff
@@ -27,6 +27,16 @@
  */
 class AllCommand extends BakeCommand
 {
+    /**
+     * All commands to call.
+     *
+     * @var array
```

Comment:
string[] then

Reply:
Sorry, I don't understand? Are you saying it should be a string, not an array?

Reply:
An array of strings. See my code fenced example now.

Reply:
```suggestion
     * @var string[]
```

Reply:
Tests are failing with `Error: Class 'ModelCommand' not found`. I'm not sure how to fix that tbh. I tried adding a `use Bake\Command\ModelCommand;` declaration to the test but that didn't help. 
```
Bake\Test\TestCase\Command\AllCommandTest::testExecuteEverything
Error: Class 'ModelCommand' not found
bake/tests/TestCase/Command/AllCommandTest.php:138
```